<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CIhm</class>
 <widget class="QMainWindow" name="CIhm">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>835</width>
    <height>588</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Serveur de test de PMV (PhA v1.1)</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextEdit" name="teTexte">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>411</width>
      <height>361</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>807</width>
      <height>120</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Ecoute sur le port </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Port : </string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lePort">
         <property name="text">
          <string>4444</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Maximum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>200</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pbStart">
         <property name="text">
          <string>START</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0,1,0">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Emettre : </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="leTexte">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>450</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>{&quot;command&quot;: &quot;btnState&quot;,   &quot;data&quot;: { &quot;btnSess&quot;: 0, &quot;btnPrep&quot;: 1, &quot;btnAVM&quot;: 0, &quot;btnReady&quot;: 0,&quot;btnGo&quot;: 0 } }</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cbSautLigne">
         <property name="text">
          <string>Ajout saut de ligne</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbEnvoyer">
         <property name="text">
          <string>Envoyer</string>
         </property>
         <property name="default">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="lineWidth">
          <number>1</number>
         </property>
         <property name="text">
          <string>Liste des clients connect√©s : </string>
         </property>
         <property name="scaledContents">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cbListe">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="gbState">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>150</y>
      <width>381</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>Etat des boutons</string>
    </property>
    <widget class="QPushButton" name="pbBtnState">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>90</y>
       <width>88</width>
       <height>34</height>
      </rect>
     </property>
     <property name="text">
      <string>btnState</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rbSess">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>105</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>btnSess</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="rbPrep">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>105</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>btnPrep</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rbAVM">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>105</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>btnAVM</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rbReady">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>30</y>
       <width>105</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>btnReady</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rbGo">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>60</y>
       <width>105</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>btnGo</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gbRunner">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>300</y>
      <width>381</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title">
     <string>Runner</string>
    </property>
    <widget class="QPushButton" name="pbTransferRunner">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>130</y>
       <width>111</width>
       <height>34</height>
      </rect>
     </property>
     <property name="text">
      <string>transferRunner</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pbGetControl">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>141</width>
       <height>34</height>
      </rect>
     </property>
     <property name="text">
      <string>Reprendre la main</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>835</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
